src/coreclr/jit/disasm.cpp

```diff
@@ -37,6 +38,8 @@
 
 /*****************************************************************************/
 
+#ifdef USE_MSVCDIS
```

Comment:
Do we ever use MSVCDIS? Can we remove all of this?

Reply:
We currently don't, but we could resurrect the code and use it. It shouldn't be too hard to do so, and there might be benefits. The only downside is it's not open source. So I'd prefer to just leave it.